history/titles/000_wc_other_titles.txt

```diff
@@ -372,6 +372,7 @@ d_witherbark = {
 	503.7.20 = { holder = amani20 } #Ugononbu[8054]
 	539.1.22 = { holder = amani21 } #Miguza[8054]
 	573.1.1={holder=54000} #Zalas [8150]
+	573.1.1 = {liege = k_zulaman}  #united under Zul'jin
```

Comment:
They should become free after Zul'jin disappears.

Reply:
It is now so.